Cargo.toml

```diff
@@ -78,7 +78,10 @@ license = "Apache-2.0"
 repository = "https://github.com/risingwavelabs/risingwave"
 
 [workspace.dependencies]
-foyer = { version = "0.10.4", features = ["nightly", "mtrace"] }
+foyer = { git = "https://github.com/foyer-rs/foyer", rev = "7eeb783", features = [
```

Comment:
There's no release on crates.io?

Reply:
https://github.com/risingwavelabs/risingwave/pull/18313#discussion_r1735826769

Here. 🤣 I'm waiting for all bugs are fixed.